gspread/models.py

```diff
@@ -986,6 +986,65 @@ def delete_row(self, index):
         }
 
         return self.spreadsheet.batch_update(body)
+    def add_protect_ranges(self, start_row_index,end_row_index,start_column_index,end_column_index, 
```

Comment:
For consistency, using the `cast_to_a1_notation` decorator might be better. This is currently used in a few other places and allows end users to either use A1 range notation or start/end indexes. For example, look at the `range` function.